builtin/mainmenu/content/contentdb.lua

```diff
@@ -427,11 +428,12 @@ function contentdb.set_packages_from_api(packages)
 				-- We currently don't support name changing
 				local suffix = "/" .. package.name
 				if alias:sub(-#suffix) == suffix then
-					contentdb.aliases[alias:lower()] = package.id
+					contentdb.aliases[strip_game_suffix(packages.type, alias:lower())] = package.id
 				end
 			end
 		end
 	end
+	print(dump(contentdb.aliases))
```

Comment:
Helpful for testing; but please remove it before merge.

Reply:
Oops, thanks for spotting it!